src/main/kotlin/javabot/operations/KarmaOperation.kt

```diff
@@ -88,10 +88,10 @@ class KarmaOperation @Inject constructor(bot: Javabot, adminDao: AdminDao, var d
                         }
                         increment = false
                     }
-                    var karma: Karma? = dao.find(nick)
+                    var karma: Karma? = dao.find(nick.toLowerCase())
```

Comment:
this search is already case insensitive.  it, in fact, upper cases the name before querying.